crates/utils/src/tracing/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod grpc;
```

Comment:
I've added this in anticipation of splitting the `logging.rs` into here with `otel` and `stdout` modules.